### ISSUE TITLE
New version

### ISSUE THREAD

**Issue by shimbala** [trippo/ResponsiveFilemanager#715]
Is there a plan for new version?

**Comment by jshster** [comment]
I don't believe there is.  I had a few brief interactions with the original project owner but nothing for a long time and there have been no replies.    If you download the latest update (not the release) you'll see there are a heap of improvements people have made.
